credentials/test/gen-test-attestation-certs.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+#
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Script that can be used to generate CHIP device attestation certificates
+# for testing purposes.
+#
+# Usage:
+#     ./gen-test-attestation-certs.sh
+# or from the root directory:
+#     ./credentials/test/gen-test-attestation-certs.sh
+#
+# The result will be stored in:
+#     credentials/test/attestation
+#
+# If the intention is to re-generate new set of attestation certificates that
+# already present then it is recommended remove current certificates:
+#     rm credentials/test/attestation/*
+#
+
+here=${0%/*}
+
+set -e
+
+CHIP_ROOT=$(cd "$here/../.." && pwd)
+
+dest_dir="$CHIP_ROOT/credentials/test/attestation"
+
+mkdir -p "$dest_dir"
+
+chip_cert_tool="$CHIP_ROOT/out/debug/linux_x64_gcc/obj/src/tools/chip-cert/bin/chip-cert"
```

Comment:
This doesn't work on Mac, obviously, or Windows.

It also relies on some sort of build step (which is not documented anywhere here) being done to create that binary.

What we should do is document what that build step is and then use something like `find $CHIP_ROOT -path "*/bin/chip-cert"` or `find $CHIP_ROOT -path "*gcc*bin/chip-cert"` or something like that, depending on what the build step is, to find the relevant binaries.

Reply:
Please don't use find for this as it could easily find the wrong (e.g. stale) binary. Recommend passing it as an argument.

Reply:
Updated to pass `chip-cert` tool binary as an input argument.